csrc/ir/interface_nodes.h

```diff
@@ -545,7 +545,6 @@ class NVF_API TensorView : public Val {
     return !mesh_.vector().empty();
   }
 
- protected:
```

Comment:
Is this for `setDomain`? Is it used?

Reply:
Not any more after @jacobhinkle's change.